src/server.ts

```diff
@@ -29,11 +31,16 @@ import { Server } from 'http';
 // we import here to allow configs to fully load
 import * as bootstrap from './bootstrap';
 import app from './app';
+import { GlobalGqlContext } from './app';
 import { database, up } from 'migrate-mongo';
 
-import { ApolloServer } from '@apollo/server';
-import { startStandaloneServer } from '@apollo/server/standalone';
+import { ApolloServer, ContextFunction } from '@apollo/server';
+import {
+  StandaloneServerContextFunctionArgument,
+  startStandaloneServer,
+} from '@apollo/server/standalone';
 import schema from './schemas/index';
+// import { EgoJwtData } from '@icgc-argo/ego-token-utils/dist/common';
```

Comment:
Can remove commented code?
I would include EgoJwtData with the GlobalGqlContext Type, which should have `userJwtData: EgoJwtData | null;`

Reply:
not sure why but argo clinical running locally is not able to fetch the type EgoJwtData from 
@icgc-argo/ego-token-utils/dist/common.

Reply:
`@icgc-argo/ego-token-utils` needs to be installed

Reply:
Oh yes! Done. Thanks.